README.md

```diff
@@ -181,5 +181,22 @@ For detailed instructions, see [building](doc/BUILDING.md).
 - See the [contribution guide](doc/contributing/PROCESS.md)
 - Join our [working group](https://github.com/OpenAssetIO/OpenAssetIO-WG) meetings
 
+> **Warning**
+> OpenAssetIO is a project used in commercial workflows, and due to this
+> need to work stably and reliably, it is a project with rigid
+> standards.
+>
+> Combined with its inherently abstract nature, it is not a good project
+> to dip your toes lightly into. Beyond minor documentation tweaks, it
+> is difficult to find work appropriate for less experienced
+> implementors.
+>
+> If you have a prevailing interest in the project, and especially if
+> you have VFX industry experience, we are delighted to meet you, but
+> otherwise you may want to consider alternate projects.
+>
+> When planning contributions, you are encouraged to reach out to
+> maintainers directly, as out-of-the-blue PR's are unlikely to succeed.
```

Comment:
[fix] Maybe a little less "go away"?

```suggestion
> If you have a prevailing interest in the project, please reach our and start a 
> thread in the Discussions section. This helps us coordinate work and 
> the approach to ensure we maintain the standard our users demand of the 
> project. Aside from to help fix our glorious range documentation typos, 
> out-of-the-blue PR's without coordination are unlikely to be accepted.

Reply:
I'll admit that was sort of what I was going for, but I see the point.

Edited your suggestion very slightly to remove the whimsy, this is a serious section for serious people doing serious things seriously. I also wanted to leave the door open for obvious non-doc things too, like updating error strings or something like that.

https://github.com/OpenAssetIO/OpenAssetIO/pull/1188/commits/a95f283d3cfe6597ccce817c0473339e75558e08

Reply:
Noice. Thanks 🙏 sorry for futzing.